runtimes/eden/src/pallets_util.rs

```diff
@@ -163,12 +163,18 @@ impl Default for SponsorshipType {
 	}
 }
 
+parameter_types! {
+	pub const PotDeposit: Balance = 1000 * constants::NODL;
+	pub const RegisterLimit: u32 = 200;
```

Comment:
This needs to be substantially more. We are targeting millions of users for our upcoming app.

Reply:
How about a per user deposit instead. So we don't set any limitations on the number?